source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -464,3 +464,10 @@ def event_UIA_notification(
 		for line in displayString.splitlines():
 			if line and not line.isspace():  # Don't say "blank" during autoread
 				speech.speakText(line)
+
+#: WinTerminalUIA is kept for backwards compatibility.
+WinTerminalUIA = (
```

Comment:
this should use `_getattr__` as per `devDocs/deprecations.md`

Reply:
Done.